src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -75,6 +75,11 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
      */
     CHIP_ERROR UpdateCommissioningParameters(const chip::Controller::CommissioningParameters & params);
 
+    /**
+     * Update the CommissioningParameters used by the active device commissioner and set the NOC Chain
+     */
+    CHIP_ERROR SetNOCChain(const chip::Controller::CommissioningParameters & params);
```

Comment:
This is a misnomer: setting the params is not setting the NOCChain. Suggest renaminging

Reply:
Removed this method